admin/views/service-manager/service-manager-home.ejs

```diff
@@ -11,13 +11,17 @@
         <%- partial('partials/_session_expiry') %>
 
         <ol class="list">
+            <!--<li>-->
```

Comment:
Is this being left in for a reason?

Reply:
Actually yes. I want to keep the existing functionality until the new check window changes are fully implemented.
I could uncomment the existing link and keep both until the 2nd one is done.
@GuyHarwood  @jon-shipley what would be the best course of action?

Reply:
That's fine, as long as there is a reason.

Reply:
@GeorgeChatzigiannis as discussed - please use a feature toggle that defaults to the old implementation until such time that the new one is usable.  Ensure the new version is accessible on via direct URL so it can be viewed if necessary.

Reply:
@GuyHarwood @jon-shipley updated with feature toggle as discussed